docs/overview/README.md

```diff
@@ -1,22 +1,51 @@
 # Overview
 
-SkillTree is an innovative approach to implementing application training.  This service reduces an application's training learning curve and drastically reduces the cost burden for providing comprehensive training. 
+SkillTree is a micro-learning gamification platform. 
+It is an innovative approach for adding *self-directed* gamified training to **an existing web-based application** (or a new application if you are building one now)!
 
-::: tip Note
-A Gamification based approach significantly simplifies the training process while making the process more enjoyable for the user community.
+::: tip Fun Fact
+We here at SkillTree believe that a Gamification based approach significantly simplifies the training process while making the process more enjoyable for the user community.
+:::
+
+Gamification training empower users to master features in a fun a way.
+- Gamification strategy setups friendly competition which provides strong motivation to continue learning; the motivation comes from points being awarded to users for performing desired actions within an application; other examples of motivators are ability to achieve the next level or earn a badge while competing against your peers.
+- Applications that implement Gamificaton for training see their users start using a wider range of available features in a more sophisticated fashion.
+- The strategy empower continued user engagement and it promotes local experts. Which at the end leads to fewer training sessions which then equates to significantly less support costs.
```

Comment:
change to: "This strategy encourages continued user engagement. Local Experts lead to a reduction in support costs by reducing the need for formal training sessions."